adminSiteClient/GdocsSettingsContentField.tsx

```diff
@@ -45,8 +48,8 @@ export const GdocsSettingsContentField = ({
                     errorType: error?.type,
                 })}
             </div>
-
             <GdocsErrorHelp error={error} />
```

Comment:
not part of this PR at all, but I'm a bit confused about the `GdocsErrorHelp` block.
That it can _either_ show an error message _or_ a help block, and never both, is a bit weird to me.